src/services/restClient.ts

```diff
@@ -0,0 +1,34 @@
+const defaultHeader = { "Content-type": "application/json" };
+
+const defaultOptions = { method: "GET", headers: defaultHeader };
+
+const _fetch = async (url: string, options: RequestInit = defaultOptions) => {
+  return fetch(url, options);
+};
+
+export const get = async (url: string, data: any) => {
+  try {
+    if(data) {
+      url += setQueryParams(data)
+    }
+    return await _fetch(url, {
+      ...defaultOptions
+    });
+  } catch (e) {
+    console.error(`Failed to fetch from ${url}`, e);
+  }
+};
+
+const setQueryParams = (params: any) => '?' + Object.keys(params).map(param => `${param}=${params[param]}`).join('&')
```

Comment:
I think we need to perform URL encoding when generating query params.

Is this API available in the browser: https://nodejs.org/api/querystring.html ?

Reply:
you're right, thanks, done.

Only difference is using encodeURIComponent instead of querystring package, since we'll be in the browser instead of running on node.